tsconfig.json

```diff
@@ -15,8 +15,10 @@
     "baseUrl": ".",
     "lib": ["esnext", "dom", "dom.iterable", "scripthost"],
     "types": ["vite/client", "node"],
-    "plugins": [{ "name": "@vuedx/typescript-plugin-vue" }],
     "typeRoots": ["src/types", "node_modules/@types"],
+    "paths": {
```

Comment:
Since it is used only 1-5 times, please remove it. Brings unnecessary complexity and inconsistency to the project :)

Reply:
Five times in vue files, I answered partly in https://github.com/MORE-Platform/more-studymanager-frontend/pull/585#discussion_r2053564478. But, why do you think this TS feature, shorter, not-relative paths + managed by TS are "unnecessary complexity and inconsistency"? Relative paths can be
```
./generated-sources
../generated-sources
../../generated-sources
.../../generated-sources
```
depending on which dir in the dir tree those imports are used. Now, all generated source imports are always in the code `@/gs`. I find exactly this less complex and more consistent to be honest.

Reply:
Correction, we have now 50 occurrences where '@gs' is used. My replace regex was malformed

Reply:
My point, you introduced something new but only used it a few times. Brings inconsistency to the application. So I'd rather have no alias but clear path imports, or as you did, change all occurrences of the import, so its consistent again. Since we wanted to avoid working on this PR too much, I opted for the first option. 
Since you changed it to the second option, its fine and in a consistent state again, but it's now much more time consuming.

Reply:
I see, sorry to cause 1) more lines to review and 2) that I diverted from the original goal, just updating deps, in the past it fit there well, but now I fully agree with you. I will do another issue in the future for the "boy scout rule" and its application and really try to focus solely on the current one